src/app/righthand-panel.js

```diff
@@ -118,7 +121,9 @@ function RighthandPanel (container, appAPI, events, opts) {
   })
 
   if (appAPI.config.exists(EDITOR_WINDOW_SIZE)) {
-    self._api.setEditorSize(appAPI.config.get(EDITOR_WINDOW_SIZE))
+    setTimeout(function () {
```

Comment:
yes this is all temporary changes that will be removed with more refactoring.
The problem is the `app.init()` function, which is called after `app.render()` in `index.js` and enables to not use `setTimeout`.

Anyway, there should neither be a `setTimeout` nor `app.init()` function, but fixing this everywhere has a lot of "ripple effects" so i would rather slowly refactor all of this and add here and there "temporary hacks" that make a pull request work, but prevent that a single pull request to become very large.